MACHINE-LEARNING-SERVICE/src/models/objectML.js

```diff
@@ -1,14 +1,28 @@
+/*
+@ObjectML.js Copyright (c) 2022 Jalasoft
+CI 26 Sur #48-41, Ayurá Center, Edificio Unión № 1376, Medellín, Colombia
+All rights reserved
+This software is the confidential and proprietary information of
+Jalasoft Confidential Information You shall not
+disclose such Confidential Information and shall use it only in
+accordance with the terms of the license agreement you entered into
+with Jalasoft.
+*/
+
 const cocoSsd = require('@tensorflow-models/coco-ssd');
 const tf = require('@tensorflow/tfjs-node');
 const fs = require('fs').promises;
 
+// Represents the object recognition system
 class ObjectML{
+
   constructor (image, percentage, objectRequired){
     this.image = image;
     this.percentage = percentage;
     this.objectRequired = objectRequired;
   }
-
+ 
+  // Method predict() allows to load the model and decode the image in order to make a detection of the desired object.
```

Comment:
I think that "Method predict()" is not necessary. You can start with "Allows to load ..."